README.md

```diff
@@ -18,11 +18,63 @@ repositories.
 Run `borges --help` to get help about the main commands (producer and consumer)
 and their options.
 
+## Setting up borges configuration
+
+Borges needs a database and a broker to do its job. It will connect to a postgres database by default
+and use rabbitMQ as default too. If you want to set up some configuration
+about the database, you can do it through the following environment variables:
+* `CONFIG_DBUSER`, by default: `testing`
+* `CONFIG_DBPASS`, by default: `testing`
+* `CONFIG_DBHOST`, by default: `0.0.0.0`
+* `CONFIG_DBPORT`, by default: `5432`
+* `CONFIG_DBNAME`, by default: `testing`
+* `CONFIG_DBSSLMODE`, by default: `disable`
+* `CONFIG_DBAPPNAME`, by default: ``
+* `CONFIG_DBTIMEOUT`, by default: `30s`
+
+To config other imprtant settings you should use:
+* `CONFIG_TEMP_DIR`, by default: `/tmp/sourced`
+* `CONFIG_BROKER`, by default: `amqp://localhost:5672`
+* `CONFIG_ROOT_REPOSITORIES_DIR`,  (where to leave siva files) by default: `/tmp/root-repositories`
+* `CONFIG_LOCKING`, by default: `local:`, other options: `etcd:`
+* `CONFIG_HDFS`, (host:port) by default: `""`
```

Comment:
Can you add: `If this property is not provided, all root repositories will be stored into the local fs`